readme.md

```diff
@@ -119,18 +119,38 @@ To tail the logs from the Docker containers in realtime, run:
 docker compose logs -f
 ```
 
-## Setup with Virtualenv
+## Setup with venv
 
-You can run the Wagtail demo locally without setting up Vagrant or Docker and simply use Virtualenv, which is the [recommended installation approach](https://docs.djangoproject.com/en/3.2/topics/install/#install-the-django-code) for Django itself.
+You can run the Wagtail demo locally without setting up Vagrant or Docker and simply use venv, which is the [recommended installation approach](https://docs.djangoproject.com/en/5.1/topics/install/#installing-an-official-release-with-pip) for Django itself.
 
 #### Dependencies
 
 - Python 3.10, 3.11 or 3.12
-- [Virtualenv](https://virtualenv.pypa.io/en/stable/installation.html)
+- [Venv](https://docs.python.org/3/library/venv.html)
 - [VirtualenvWrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html) (optional)
 
 ### Installation
 
+On GNU/Linux or MacOS (bash):
+```bash
+python -m venv wagtailbakerydemo
+source wagtailbakerydemo/bin/activate
+```
+
+On Windows (cmd.exe), run the following commands:
+```bash
+wagtailbakerydemo\Scripts\activate.bat
+# if wagtailbakerydemo\Scripts\activate.bat doesn't work, run:
+wagtailbakerydemo\Scripts\Activate.ps1
+
+```
+Note: On Ubuntu systems, the [Venv](https://docs.python.org/3/library/venv.html) package may not be installed by default. To install it, use the following command:
+```bash
+sudo apt install python3-venv
+``` 
```

Comment:
```suggestion
```

I see this is absent from the Django docs, which we reference as our reason for using this setup. And our docs above mention "Developers more familiar with venv" when directing people to this section. So I think we can skip this.